src/Microsoft.AspNetCore.SignalR/Internal/ObjectMethodExecutor.cs

```diff
@@ -0,0 +1,262 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Internal;
+
+namespace Microsoft.AspNetCore.SignalR.Internal
+{
+    internal class ObjectMethodExecutor
```

Comment:
Can we add tests for this class? I don't think our tests cover all the cases like static vs. instance methods, Task vs Task<T> vs, void return types, default parameter values etc.